.github/workflows/tests.yml

```diff
@@ -445,6 +445,155 @@ jobs:
+        with:
+          name: system-test-deps
+          merge-multiple: true
+          path: /home/runner/go/bin
+
+      - name: Set exec perms on LXD binaries
+        run: |
+          set -eux
+          ls -lR /home/runner/go/bin/
+          chmod uog+x /home/runner/go/bin/*
+
+      - name: Download docs (for the objects.inv.txt file to be available for the e2e tests)
+        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4.1.8
+        with:
+          name: documentation
+          merge-multiple: true
+          path: doc/_build
+
+      - name: Make GOCOVERDIR
+        run: |
+          set -eux
+          mkdir -p "${GOCOVERDIR}"
+        if: env.GOCOVERDIR != ''
+
+      - name: Run LXD daemon
+        run: |
+          set -eux
+          echo "root:1000000:1000000000" | sudo tee /etc/subuid /etc/subgid
+          export LXD_DOCUMENTATION="/home/runner/work/lxd/lxd/doc/_build/"
+          export PATH="/home/runner/go/bin:$PATH"
+          sudo rm -rf /var/lib/lxd
+          sudo --preserve-env=PATH,GOPATH,GOCOVERDIR,LD_LIBRARY_PATH,LXD_DOCUMENTATION $(go env GOPATH)/bin/lxd --group sudo &
+
+      - name: Run LXD-UI
+        env:
+          ENVIRONMENT: devel
+          PORT: 8407
+          LXD_UI_BACKEND_IP: 172.17.0.1
+        run: |
+          set -eux
+          cd lxd-ui
+          dotrun &
+          curl --head --fail --retry-delay 2 --retry 100 --retry-connrefused --insecure https://localhost:${{ env.PORT }}
+
+      - name: Setup LXD
+        shell: bash
+        run: |
+          set -eux
+          export PATH="/home/runner/go/bin:$PATH"
+          sudo -E LXD_DIR=/var/lib/lxd lxc storage create default zfs
+          sudo -E LXD_DIR=/var/lib/lxd lxc profile device add default root disk path=/ pool=default
+          sudo -E LXD_DIR=/var/lib/lxd lxc network create local-network
+          sudo -E LXD_DIR=/var/lib/lxd lxc profile device add default eth0 nic network=local-network
+          sudo -E LXD_DIR=/var/lib/lxd lxc config set core.https_address "[::]:8443"
+          sudo -E LXD_DIR=/var/lib/lxd lxc config set cluster.https_address "127.0.0.1"
+          sudo -E LXD_DIR=/var/lib/lxd lxc cluster enable local
+          sudo -E LXD_DIR=/var/lib/lxd lxc config set user.show_permissions=true
+          sudo -E LXD_DIR=/var/lib/lxd lxc config trust add lxd-ui/keys/lxd-ui.crt
+
+      - name: Setup for LXD-UI tests
+        shell: bash
+        run: |
+          set -eux
+          sudo -E LXD_DIR=/var/lib/lxd PATH=/home/runner/go/bin:$PATH ./lxd-ui/tests/scripts/setup_test
+
+      - name: Run Playwright tests
+        run: |
+          set -eux
+          cd lxd-ui
+          sudo chown $USER -R /home/runner/.config
+          sudo chown $USER /var/lib/lxd/unix.socket
+          CI=true DISABLE_VM_TESTS=true LXD_DIR=/var/lib/lxd PATH=/home/runner/go/bin:$PATH npx playwright test --project chromium:lxd-latest-edge
```

Comment:
why dont you add `PATH=/home/runner/go/bin:$PATH` to the `env` section to save repeating it in a few places?

https://github.com/canonical/lxd/pull/14035/files#diff-1db27d93186e46d3b441ece35801b244db8ee144ff1405ca27a163bfe878957fR453

Reply:
I tried that, and I think the resolution of `$PATH` in the `env` section was not producing the same result as doing it in here.